e3db/base_crypto.py

```diff
@@ -83,3 +86,11 @@ def encrypt_file(self, plaintext_file, key):
     @classmethod
     def decrypt_file(self, encrypted_file, destination_file, key):
         pass
+
+    @classmethod
+    def hashString(self, toHash: str) -> bytes:
+        return HASHER(toHash.encode("utf-8"), encoder=nacl.encoding.RawEncoder)
+
+    @classmethod
+    def base64HashString(self, toHash: str) -> bytes:
```

Comment:
Is this return type correct?

Reply:
blake2b method definition has `:rtype: bytes`, so I believe so